README.md

```diff
@@ -26,13 +26,14 @@ The Python bindings for MaterialX are based on [PyBind11](https://github.com/pyb
 
 The MaterialX repository consists of the following folders:
 
-    documents - The MaterialX specification, developer guide, and example files.
-    source    - A cross-platform C++ library for MaterialX with Python bindings.
-                The MaterialXCore module supports the core MaterialX elements
-                and graph traversal, while the MaterialXFormat module supports
-                XML serialization.
-    python    - Support modules for MaterialX Python.
+- [documents](documents) - The MaterialX specification, developer guide, example and test suite files.
+- [source](source) - Cross-platform C++ libraries for MaterialX with Python bindings.
+- [python](python) - Support modules for MaterialX Python.
+
+### Usage
+
+Use of this code is subject to the terms of the Autodesk license agreement provided at the time of installation or download, or which otherwise accompanies this software in either electronic or hard copy form.
```

Comment:
This language was probably not intended to be merged to MaterialX master, though it makes perfect sense within the Autodesk fork.

Reply:
Yes. We'll remove this.

Reply:
This has been removed.